### DISCUSSION TITLE
Support for Flying Bear Ghost 5

### DISCUSSION THREAD

**Discussion by Scrapagnan** [OrcaSlicer/OrcaSlicer#5063] [Dev]
Hello,
Can anyone create the printer profile for Flying Bear Ghost 5?

Volume: 255x210x210
Mainboard: v5.8.5
Wi-Fi: v5.8.5
Firmaware: Marlin (idk the version), but i know that it has the firmware of the sister FBG4S.

Tell me if you need more information about the printer

Thank you very much

**Comment by MathieuIko** [comment]
Hi,
You might look at this profile by TabLAB on MakerWorld: https://makerworld.com/en/models/168431
His profile is for OrcaSlicer v1.9+, so I suppose it also works for OrcaSlicer 2.0 (I haven't tested it yet).

**Reply by vitalyu** [reply]
> Hi, You might look at this profile by TabLAB on MakerWorld: https://makerworld.com/en/models/168431 His profile is for OrcaSlicer v1.9+, so I suppose it also works for OrcaSlicer 2.0 (I haven't tested it yet).

nope, doesn't work

**Reply by LTMaster** [reply]
Have anyone solved this?

**Comment by Mykoriak** [comment]
+1

**Comment by nexurion** [comment]
+1
